cirq/contrib/two_qubit_gates/gate_compilation_test.py

```diff
@@ -0,0 +1,59 @@
+"""Tests for gate_compilation.py"""
+import numpy
+import pytest
+
+# pylint: disable=line-too-long
+from cirq import unitary, FSimGate
+from cirq.contrib.two_qubit_gates.gate_compilation import gate_product_tabulation
+from cirq.contrib.two_qubit_gates.math_utils import random_two_qubit_unitaries_and_kak_vecs, unitary_entanglement_fidelity
+
+# pylint: enable-line-too-long
+
+# coverage: ignore
```

Comment:
I don't think this should be required for tests, they should already be ignored for coverage.

Reply:
Done